### PR TITLE
fix: router type change

### PR DESCRIPTION
**Pull request by FuBaooo**
#274 

Please see here for a detailed description of the problem.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -98,30 +98,30 @@
     "html5parser": "^2.0.2",
     "jsdom": "^20.0.0",
     "kolorist": "^1.5.1",
-    "p-queue": "6.6.2",
+    "p-queue": "7.3.0",
```
Comment (antfu) [inline package.json:101]: Can you avoid bumping other deps? (we could do it in another PR)

Comment (FuBaooo) [reply package.json]: Sorry, I didn't notice this, and it has been revised.

File: package.json [hunk 2]
```diff
@@ -106,22 +106,22 @@
...
+    "eslint": "^8.21.0",
     "esno": "^0.16.3",
-    "rollup": "^2.75.7",
-    "tsup": "^6.1.2",
+    "rollup": "^2.77.2",
+    "tsup": "^6.2.1",
     "typescript": "^4.7.4",
-    "unbuild": "^0.7.4",
-    "vite": "^2.9.13",
-    "vite-plugin-pwa": "^0.12.1",
```
Comment (antfu) [inline package.json:122]: Also here.

Comment (FuBaooo) [reply package.json:122]: I think it is enough to only change the dependency of the route. I restored other dependent versions
